css/types/attr.json

```diff
@@ -0,0 +1,105 @@
+{
+  "css": {
+    "types": {
+      "attr": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/attr",
+          "description": "<code>attr()</code>",
+          "support": {
+            "webview_android": {
+              "version_added": "2.1"
```

Comment:
Please change to `true`. Android webview follows Chrome number, not Android. I'm currently unable to get an exact version number for this feature.